frontend/package.json

```diff
@@ -16,22 +16,27 @@
     "postinstall": "nuxt prepare",
     "lint": "eslint --max-warnings 0 .",
     "typecheck": "nuxt typecheck",
-    "start": "NODE_ENV=production node ."
+    "start": "NODE_ENV=production node .",
+    "generateDocs": "vite-node ./scripts/generateDocs.ts"
   },
   "devDependencies": {
-    "@huntersofbook/naive-ui-nuxt": "^0.5.1",
```

Comment:
I've downgraded it since it stopped properly working with tailwind: more info [in another project of ours](https://github.com/sidestream-tech/t4f-gxmanager/pull/52). So the suggestion is to keep it down

Reply:
Fixed via 809fc9a